src/main/java/org/opentripplanner/standalone/configure/LoadApplicationModule.java

```diff
@@ -0,0 +1,22 @@
+package org.opentripplanner.standalone.configure;
+
+import dagger.Module;
+import dagger.Provides;
+import java.io.File;
+import org.opentripplanner.standalone.config.CommandLineParameters;
+import org.opentripplanner.standalone.config.api.OtpBaseDirectory;
+
+/**
+ * Main focus is to provide bindings not provided elsewhere for the load application phase. For
+ * example binding specific config parameters to DI managed input parameters. This allows us to keep
+ * avoid passing entire configuration into modules, creating undecided dependencies.
```

Comment:
simplify javadoc comment

Reply:
I deleted this class after moving the binding.